codeshifu/fast-food-fast
# #160873536 Add shopping cart

# What does this PR do?
Creates a shopping cart for users to easily order multiple menus from the restaurant.

# How should this be manually tested?
Visit the site homepage, click add to cart on one or more menu, see the cart below the page, update cart items quantity or proceed to checkout

# Pivotal tracker story

#160873536
